### PR TITLE
Visualisation

### PR DESCRIPTION
**Pull request by lukaa12**

### CODE REVIEW

File: kentik_api_library/examples/queries_example.py [hunk 1]
```diff
@@ -115,7 +117,11 @@ def run_query_chart() -> None:
     result = client.query.chart(query_object)
 
     print("Result:")
-    print(result.__dict__)
+    # result.save_image_as("/home/lukasz/test.png")
+    img = Image.open(BytesIO(result.get_data()))
+    img.show()
+    print("File saved")
```
Comment (mateuszmidor) [inline kentik_api_library/examples/queries_example.py:123]: not sure if it is actually saved?

File: kentik_api_library/examples/queries_visualisation_example.ipynb [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.6"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```
Comment (mateuszmidor) [inline kentik_api_library/examples/queries_visualisation_example.ipynb:373]: could you please also add a second pandas example that sorts by given column and takes for example top 10 rows?
This is something they were interested in :)

File: kentik_api_library/examples/queries_example.py (hunk 1 above)
Comment (lukaa12) [reply kentik_api_library/examples/queries_example.py:123]: You are right it is not saved but i forgot to remove this print

**Review by mateuszmidor — APPROVED** [review]

**Review by TedatKentik — APPROVED** [review]
Visualizations structure

**Pull request merged**